.eslintignore

```diff
@@ -0,0 +1,11 @@
+node_modules
```

Comment:
this was to fix a precommit issue. precommit is running `eslint . --fix` but as we moved to monorepo, this is failing since `.eslintrc` got moved.